### PR TITLE
Added filter item

### PR DESCRIPTION
**Pull request by RamzesUAA**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by dreamofmaks — APPROVED** [review]

File: SoftServeCarMobile/src/activity/messages/chat/simple-message/SimpleMessageStyle.tsx [hunk 1]
```diff
@@ -37,12 +32,31 @@ const SimpleMessageStyle = StyleSheet.create({
...
+        borderWidth: 2,
+        borderBottomWidth: 2,
+        borderBottomColor: 'black',
+        paddingLeft: 5
+    },
+    textStyle:{
         fontSize: 11,
         paddingTop: 10,
-        fontFamily: "sans-serif"
+        fontFamily: 'sans-serif'
```
Comment (Reennon) [inline SoftServeCarMobile/src/activity/messages/chat/simple-message/SimpleMessageStyle.tsx:57]: Please, use font manager file!

**Review by Reennon — CHANGES_REQUESTED** [review]
Please make sure you specified fonts you've to use from Fonts.<android/ios>.tsx

File: SoftServeCarMobile/src/activity/messages/chat/simple-message/SimpleMessage.tsx [hunk 2]
```diff
@@ -1,94 +1,95 @@
...
+            .then(res => {
+                const chats = res.data;
+                console.log(chats);
+                setFilteredDataSource(chats);
+                setMasterDataSource(chats);
+            })
     }, []);
 
-    if (isLoading) {
+    const searchFilterFunction = (text) => {
```
Comment (AntonukOM) [inline SoftServeCarMobile/src/activity/messages/chat/simple-message/SimpleMessage.tsx:29]: let's call this method setSearchFilter

File: SoftServeCarMobile/src/activity/messages/chat/simple-message/SimpleMessage.tsx [hunk 3]
```diff
@@ -1,94 +1,95 @@
...
+
+    const ItemView = ({ item }) => {
         return (
-            <View>
-                <ActivityIndicator size="large" color="#5500dc" />
+            <View style={SimpleMessageStyle.main}>
+                <View style={SimpleMessageStyle.button}>
+                    <View style={{ flexDirection: 'row' }}>
+                        <Image style={SimpleMessageStyle.image}
+                            source={{ uri: 'https://cdn.pixabay.com/photo/2015/04/23/22/00/tree-736885__340.jpg' }} />
```
Comment (AntonukOM) [inline SoftServeCarMobile/src/activity/messages/chat/simple-message/SimpleMessage.tsx:65]: hardcoded resource, please move uri to const

File: SoftServeCarMobile/src/activity/messages/messages-tabs/MessagesTabs.tsx [hunk 4]
```diff
@@ -19,6 +19,11 @@ const ChatTabs = () => {
     const userServices = container.resolve(UserService);
     const [currentUser, setCurrentUser] = useState({} as User);
     const { user } = useContext(AuthContext);
+    const [isOpenFilter, setIsOpenFilter] = useState(false);
+
+    const setIsOpen = () => {
+        setIsOpenFilter(!isOpenFilter);
```
Comment (AntonukOM) [inline SoftServeCarMobile/src/activity/messages/messages-tabs/MessagesTabs.tsx:25]: we set isOpenFilter, setIsOpenFilter to false, and after that revert the value. Is it better to set the default state to true?

**Review by AntonukOM — CHANGES_REQUESTED** [review]

File: SoftServeCarMobile/src/activity/messages/messages-tabs/MessagesTabs.tsx (hunk 4 above)
Comment (RamzesUAA) [reply SoftServeCarMobile/src/activity/messages/messages-tabs/MessagesTabs.tsx:25]: I'll explain to you why I've written it in such a way in the daily.

File: SoftServeCarMobile/src/activity/messages/chat/simple-message/SimpleMessage.tsx (hunk 3 above)
Comment (RamzesUAA) [reply SoftServeCarMobile/src/activity/messages/chat/simple-message/SimpleMessage.tsx:65]: It just a temporary URL for better visibility, here I will use a component from notifications.

**Review by AntonukOM — APPROVED** [review]

**Review by Reennon — APPROVED** [review]
Nice

**Pull request merged**